### PR TITLE
Sort containers by name

### PR DESCRIPTION
**Pull request by agungyuliaji**
Small change sort containers by name, useful and neat when the containers are many

### CODE REVIEW

File: hook/index.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@
   Docker::Container.expire_cache!
   me = Docker::Container.me
   containers = Docker::Container.all - [me]
+  containers.sort_by! { |c| c.name }
```
Comment (mtsmfm) [inline hook/index.rb]: Could you sort in L38?

```ruby
containers.sort_by(&:name).flat_map do |c|
  ...
```

**Review by mtsmfm — COMMENTED** [review]
Thank you for your contribution!
I'm really glad to receive first PR for this small project😂

File: hook/index.rb (hunk 1 above)
Comment (mtsmfm) [reply hook/index.rb]: Ah, Sorry there are two loops 🙈 

Could you move this line to L18?
`containers` instance is assigned by L6 only if all networks is connected to yaichi's container.

Comment (agungyuliaji) [reply hook/index.rb]: 🙈 Fixed!

File: hook/index.rb [hunk 2]
```diff
@@ -14,6 +14,7 @@
   me = Docker::Container.me
   containers = Docker::Container.all - [me]
 end
+containers.sort_by! { |c| c.name }
```
Comment (mtsmfm) [inline hook/index.rb]: I noticed that there is no `sort_by!` on mruby 🤔 
https://ruby-doc.org/core-mruby/Enumerable.html#method-i-sort_by

`containers = containers.sort_by(&:name)` works well

Comment (mtsmfm) [reply hook/index.rb]: (I'll write document later though)
You can boot yaichi and confirm new behavior on your machine 😉 

```
$ git clone https://github.com/mtsmfm/yaichi
$ cd yaichi
$ # make sure you don't run process which use 80 port
$ docker-compose up
$ open http://localhost:80
```

Comment (agungyuliaji) [reply hook/index.rb]: Thanks!!!



Yeah actually contributed without run on my machine 🙈 , confused how to run yaichi with a github fork, so bad! 🙇

**Review by mtsmfm — APPROVED** [review]
:shipit: :shipit: :shipit:

**Comment by mtsmfm** [comment]
@agungyuliaji 
Thanks!
It will be pushed automatically within 30 min.
https://hub.docker.com/r/mtsmfm/yaichi/

**Pull request merged**